### DISCUSSION TITLE
Clase 4 y Tarea

### DISCUSSION THREAD

**Discussion by JulianBuffoni** [dds-jv/foro#360] [Consultas generales] [answered]
Buenas, tengo varias consultas:

- Está la clase 4 actualizada?
- Hay que hacer tanto QMP3 como el ejercicio de monedero?
- El ejercicio de monedero es individual?

**Comment by gastonprieto** [comment, marked as answer]
buenas @JulianBuffoni 

que curso ?

**Reply by JulianBuffoni** [reply]
Jueves Mañana

**Reply by gastonprieto** [reply]
listo @JulianBuffoni ya esta actualizado
